pkg/netpol/connlist/connlist.go

```diff
@@ -188,7 +194,8 @@ func ConnectionsListToString(conns []Peer2PeerConnection) string {
 		connLines[i] = conns[i].String()
 	}
 	sort.Strings(connLines)
-	return strings.Join(connLines, "\n")
+	newlineChar := fmt.Sprintln("")
```

Comment:
Not sure I understand the problem with using `\n`
That is a line break in all platforms but Windows

Reply:
Println is better if you care about Windows as a runtime platform